usage.go

```diff
@@ -7,6 +7,7 @@ const (
 	TASKS_COMPLETE              ExitCode = 0
 	CANT_LOAD_CONFIG            ExitCode = 64
 	CANT_INSTALL_GENERIC_WORKER ExitCode = 65
+	CANT_REMOVE_GENERIC_WORKER  ExitCode = 66
```

Comment:
It might be better to use a different exit code here, as this [used to mean something else](https://github.com/taskcluster/generic-worker/commit/d5a8dd552bed48bd174b43dec40742fc8cfeac74#diff-04c6e90faac2675aa89e2176d2eec7d8L378).